test/e2e_tests/dir_test.go

```diff
@@ -394,27 +394,29 @@ func (suite *dirTestSuite) TestDirRenameFull() {
 }
 
 func (suite *dirTestSuite) TestTarDir() {
-	dirName := suite.testPath + "/tar"
-	tarName := suite.testPath + "/tardir.tar.gz"
+	if clonePtr == "true" || clonePtr == "True" {
```

Comment:
TarDir test case works with streaming ?

Reply:
TarDir test has a git clone operation, therefore we should add the clonePtr check since stream does not support git clone.

Reply:
I have moved all these cases into single git test case for optimization. Once my pr is merged this all will be fixed.